ovs/extensions/generic/disk.py

```diff
@@ -81,6 +81,36 @@ def create_partition(disk_path, disk_size, partition_start, partition_size):
         check_output(command_2, shell=True)
 
     @staticmethod
+    def get_partitions_info(device):
+        """
+        Get partitions info from a device
+        :param device: device name: /dev/sda
+
+        :return: dict {'partition_device': {'key': 'value'}}
+        (!) return values are in B not in sectors
+        """
+        result = {}
+        command = "parted -m {0} unit B print -s".format(device)
+        logger.debug('Checking partitions with command: {0}'.format(command))
+        try:
+            output = check_output(command, shell=True).splitlines()
+        except CalledProcessError as cpe:
+            raise RuntimeError('{0} {1}'.format(cpe, cpe.output))
+        for line in output[:]:
+            output.pop(0)
```

Comment:
Remove "warnings or stderr info" before "BYT;" line.
Not parseable, so removing it.